src/lib/sponsoring/Sponsors.svelte

```diff
@@ -15,7 +15,6 @@
 		<a href={sponsor.link} rel="external">
 			<div class="box">
 				<div class="image"><img src={sponsor.image} alt={sponsor.name} /></div>
-				<div class="name"><span>{sponsor.name}</span></div>
 				{#if sponsor.sponsoring}<div class="sponsoring">{sponsor.sponsoring}</div>{/if}
 			</div>
```

Comment:
I think we had some sponsors in the past where it wasn't really clear what the name was from their logo itself. Maybe we should change the design here completely like adding a tooltip or putting the name only into the hover / focus / details part for the sponsor...?